include/net/nrf_cloud.h

```diff
@@ -198,7 +219,7 @@ enum nrf_cloud_error {
 	NRF_CLOUD_ERROR_NO_DEV_NOT_PROV		= 40412,
 	NRF_CLOUD_ERROR_NO_DEV_DISSOCIATE	= 40413,
 	NRF_CLOUD_ERROR_NO_DEV_DELETE		= 40414,
-	/** Item was not found. No error occured, the requested item simply does not exist */
+	/* Item was not found. No error occurred, the requested item simply does not exist */
```

Comment:
I guess this is not intended to be shown?

Reply:
Correct, these are documented in the cloud docs.  Just making a note here in the code.